test/controllers/beneficial.owner.type.controller.spec.ts

```diff
@@ -12,11 +12,14 @@ import * as config from "../../src/config";
 import { getApplicationData } from '../../src/utils/application.data';
 import {
   BENEFICIAL_OWNER_TYPE_PAGE_HEADING,
+  CHECK_YOUR_ANSWERS_LINK,
+  SERVICE_UNAVAILABLE,
```

Comment:
just running the tests, it has highlighted that SERVICE_UNAVAILABLE is a duplicate import